logback/src/test/java/com/linecorp/armeria/common/logback/RequestContextExportingAppenderTest.java

```diff
@@ -592,6 +595,75 @@ void testResponseCause() throws Exception {
         }
     }
 
+    @Test
+    void testMdcPropsType() throws Exception {
+        final List<ILoggingEvent> events = prepare(a -> a.addBuiltIn(BuiltInProperty.REQ_DIRECTION));
+
+        final ServiceRequestContext ctx = newServiceContext("/foo", null);
+        try (SafeCloseable ignored = ctx.push()) {
+            final ILoggingEvent e = log(events);
+            final Map<String, String> mdc = e.getMDCPropertyMap();
+            assertThat(mdc.getClass().getName())
+                    .isEqualTo("com.linecorp.armeria.common.logging.RequestContextExporter$State");
+        }
+    }
+
+    @Test
+    void testMdcPropsTypeForSocketAppenderWithoutProps() throws Exception {
+        final SocketAppender sa = prepareSocketAppender();
+        final ClientRequestContext ctx = newClientContext("/bar", null);
+        try (SafeCloseable ignored = ctx.push()) {
+            final Integer value = ThreadLocalRandom.current().nextInt();
+            testLogger.trace("{}", value);
+
+            final ArgumentCaptor<ILoggingEvent> eventCaptor = ArgumentCaptor.forClass(ILoggingEvent.class);
+            verify(sa).doAppend(eventCaptor.capture());
+
+            final Map<String, String> mdc = eventCaptor.getValue().getMDCPropertyMap();
+            assertThat(mdc.getClass().getName()).startsWith("java.util");
+            assertThat(mdc).isEmpty();
+        }
+    }
+
+    @Test
+    void testMdcPropsTypeForSocketAppenderWithProps() throws Exception {
+        final SocketAppender sa = prepareSocketAppender(a -> a.setExport("test-prop=req.headers.user-agent"));
+        final ClientRequestContext ctx = newClientContext("/bar", null);
+        try (SafeCloseable ignored = ctx.push()) {
+            final Integer value = ThreadLocalRandom.current().nextInt();
+            testLogger.trace("{}", value);
+
+            final ArgumentCaptor<ILoggingEvent> eventCaptor = ArgumentCaptor.forClass(ILoggingEvent.class);
+            verify(sa).doAppend(eventCaptor.capture());
+
+            final Map<String, String> mdc = eventCaptor.getValue().getMDCPropertyMap();
+            assertThat(mdc.getClass().getName()).startsWith("java.util");
```

Comment:
How about..
```suggestion
            assertThat(mdc).isInstanceOf(HashMap.class);
```
to make it clearer?

Reply:
ok